src/controllers/handlePointerDown.ts

```diff
@@ -7,12 +7,23 @@ import { updateStoreWithApiAndEventHandler } from "../utils/updateStoreWithApiAn
 import { ReactGridStore } from "../types/ReactGridStore.ts";
 
 export const handlePointerDown = (event: React.PointerEvent<HTMLDivElement>, storeApi: StoreApi<ReactGridStore>) => {
+  const usedTouch = event.pointerType !== "mouse"; // * keep in mind there is also "pen" pointerType
+
+  let holdTimeoutId: NodeJS.Timeout;
+
+  if (event.button === 2 || usedTouch) {
```

Comment:
What is it?

Reply:
If the event is a right-click or a touch event, a timeout is set to trigger. ```holdTimeoutId``` is used to clear the timeout if a pointer up event occurs before 500 ms, so the hold handler function will not be called

Reply:
Okay. Can you create some sort of constant assertion or simply a normal const that will describe `event.button === 2`, so the code will be easier to understand?